client/my-sites/plugins-wpcom/business-plugins-panel.jsx

```diff
@@ -9,7 +9,8 @@ import BusinessPlugin from './plugin-types/business-plugin';
 const defaultPlugins = [
 	{
 		name: 'Google Analytics',
-		supportLink: 'https://en.support.wordpress.com/google-analytics/',
+		featureDescriptionLink: '/plans/features/google-analytics',
+		linkTarget: '_self',
```

Comment:
do we really need this? can't we infer that it's a Calypso-internal link and automatically not include `target="_blank"` for those but leave them for external links?